.github/workflows/exit-pre-mode-on-comment.yml

```diff
@@ -0,0 +1,50 @@
+name: Exit pre-release
+
+on:
+  workflow_dispatch:
+  issue_comment:
+    types: [created]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  exit-pre-release:
+    name: Exit changesets pre-release mode when !exit-rc comment created
+    if: ${{ github.event.comment.body == '!exit-rc' && github.event.issue.pull_request }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".node-version"
+          cache: "yarn"
+      - name: Install dependencies
+        run: yarn --immutable
+      - name: resolve pr refs
+        id: refs
+        uses: eficode/resolve-pr-refs@main
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ steps.refs.outputs.head_ref }}
+      - name: set up env branch name var
+        run: echo "BRANCH_NAME=$(echo $(git branch --show-current))" >> $GITHUB_ENV
+      - name: Exit pre release mode for changesets
+        if: ${{ startsWith( env.BRANCH_NAME, 'v2-release' ) }}
```

Comment:
Maybe we could try to write a condition like [this](https://github.com/vuestorefront/storefront-ui/pull/2797/files#diff-4d183e7fe30b407ed91c82a3f4d3d3d51eef369bd06fe0a180bd04cf74f4e50bR14)?

Reply:
`steps.refs.outputs.head_ref` returns SHA of latest commit pushed into branch. And I can't get branch name with any `github` event